presubmits.sh

```diff
@@ -1,5 +1,15 @@
 #!/bin/bash
 
+function run-cmd {
```

Comment:
Can we use `set -e`

https://stackoverflow.com/questions/3474526/stop-on-first-error

Reply:
Good idea, done.